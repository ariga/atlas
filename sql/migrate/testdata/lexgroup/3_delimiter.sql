-- delimiter should not conflict with compound statements scanning.

create function `add2` (a int, b int) returns int deterministic no sql return a + b;
create function `add3` (a int, b int, c int) returns int deterministic no sql return a + b + c;
delimiter |
-- error 1418
create function fn1(x int) returns int deterministic
begin
       insert into t1 values (x);
       return x+2;
end|
create function fn2(x int) returns int deterministic
begin
       insert into t1 values (x);
       return x+2;
end|
delimiter ;
create function `add4` (a int, b int, c int, d int) returns int deterministic no sql return a + b + c + d;