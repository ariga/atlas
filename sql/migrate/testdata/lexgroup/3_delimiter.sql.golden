create function `add2` (a int, b int) returns int deterministic no sql return a + b;
-- end --
create function `add3` (a int, b int, c int) returns int deterministic no sql return a + b + c;
-- end --
create function fn1(x int) returns int deterministic
begin
       insert into t1 values (x);
       return x+2;
end
-- end --
create function fn2(x int) returns int deterministic
begin
       insert into t1 values (x);
       return x+2;
end
-- end --
create function `add4` (a int, b int, c int, d int) returns int deterministic no sql return a + b + c + d;